lib/notification-issue.coffee

```diff
@@ -174,12 +174,13 @@ class NotificationIssue
     return packageName if packageName
 
   getPackageName: ->
+    message = @notification.getMessage()
     options = @notification.getOptions()
-    return unless options.stack? or options.detail?
+    return unless message? or options.stack? or options.detail?
 
     packagePaths = @getPackagePathsByPackageName()
     for packageName, packagePath of packagePaths
-      if packagePath.indexOf('.atom/dev/packages') > -1 or packagePath.indexOf('.atom/packages') > -1
+      if packagePath.indexOf(path.join('.atom', 'dev', 'packages')) > -1 or packagePath.indexOf(path.join('.atom', 'packages')) > -1
```

Comment:
I think this still might be incorrect...@kevinsawicki if someone sets their `ATOM_HOME` to something like `/Users/user/ATOM`, then will this check fail?  If so, how would I fix that?  By reading `process.env.ATOM_HOME` and then regexing it to capture the last folder path?